recruiter/tests.py

```diff
@@ -276,3 +286,25 @@ def test_update_recruiter_account_settings_with_valid_data(example_recruiter, ex
     assert example_recruiter_from_db.phone_number == new_phone_number
     # Test that untouched data hasn't changed
     assert example_recruiter_from_db.company == example_recruiter.company
+
+
+@pytest.mark.django_db
+def test_add_new_comapny_form_loads_correctly(valid_company_data, client):
+    response = client.post("/companyRegister/")
+    assert response.status_code == 200
+    form = response.context["form"]
+    form_initial_data = response.context["form"].initial
+    assert isinstance(form, CompanyRegistrationForm)
+    assert all(form_initial_data[key] == valid_company_data[key] for key in form_initial_data)
```

Comment:
missing check for the form actually being displayed (the right template is used)
not sure why I still need to comment on this over and over again....

Reply:
about this, I think I understand what you saying. and I added a check for this:
`assertTemplateUsed(response, 'recruiter/registerCompany.html')`
is this ok?